grammars/level1.lark

```diff
@@ -63,6 +63,7 @@ NAME: LETTER_OR_UNDERSCORE LETTER_OR_NUMERAL*
 
 LETTER_OR_UNDERSCORE: /[\p{Lu}\p{Ll}\p{Lt}\p{Lm}\p{Lo}\p{Nl}_]+/
 LETTER_OR_NUMERAL: LETTER_OR_UNDERSCORE | /[\p{Mn}\p{Mc}\p{Nd}\p{Pc}·]+/
+SMALL_LETTER_OR_NUMERAL: /[a-z]/ | /[0-9]/
```

Comment:
Good question! I have not included those because of the following reasons:

1. Pygame will register SHIFT+[the letter] when using this, this also requires the registration of this in the PyGame game loop, which will probably complicate things for now. 
2.  Technically, the letter "A" for example is a combination of SHIFT+a. It would be cool, I think, to make "and" possible with keyboard presses in the near future. This way the users can make there own shortcuts and allowing A breaks this shortcut idea.

Let me know what you think @jpelay!